quota/etcd/quotaapi/quota_server_test.go

```diff
@@ -436,12 +436,12 @@ func TestServer_UpdateConfig_ConcurrentUpdates(t *testing.T) {
 
 					want.CurrentTokens = got.CurrentTokens // Not important for this test
 					want.MaxTokens = tokens
-					if !proto.Equal(got, &want) {
+					if !proto.Equal(got, want) {
 						diff := cmp.Diff(got, &want, cmp.Comparer(proto.Equal))
 						t.Errorf("%v: post-UpdateConfig() diff (-got +want):\n%v", want.Name, diff)
 					}
 				}
-			}(num, *cfg)
+			}(num, proto.Clone(cfg).(*quotapb.Config))
```

Comment:
Not sure how this got missed when I went looking for them last year.

Reply:
They are easy to miss